public/scss/_cardicons-sprite.scss

```diff
@@ -1,5 +1,5 @@
 .card-icon {
-	background: url("#{$image-path}cardicons-sprite.svg") no-repeat;
+	background: url(../img/cardicons-sprite.svg) no-repeat;
```

Comment:
Ah interesting. All this CSS is generated by the grunt spite generator. I'll need to look at tweaking that. The only thing that might be impacted is email.